bears/general/LicenseCheckBear.py

```diff
@@ -0,0 +1,45 @@
+import re
+
+from coalib.bearlib.abstractions.Linter import linter
+from dependency_management.requirements.DistributionRequirement import (
+    DistributionRequirement)
+
+
+@linter(executable='licensecheck',
+        use_stdout=True,
+        use_stderr=True)
+class LicenseCheckBear:
+    """
+    Attempts to check the given file for a license, by searching the start
+    of the file for text belonging to various licenses.
+    """
+    LANGUAGES = {'All'}
+    REQUIREMENTS = {DistributionRequirement(apt_get='devscripts')}
+    AUTHORS = {'The coala developers'}
+    AUTHORS_EMAILS = {'coala-devel@googlegroups.com'}
+    LICENSE = 'AGPL-3.0'
+    CAN_DETECT = {'License', 'Copyright Notice'}
+
+    _output_regex = re.compile(
+        r'(?P<message>\S+.*)')
+
+    @staticmethod
+    def create_arguments(filename, file, config_file,
+                         licensecheck_lines: int=60):
+        """
+        :param licensecheck_lines:
+            Specify how many lines of the file header should be parsed
+            for license information.
+        """
+
+        args = ('--lines', str(licensecheck_lines))
+        return args + (filename,)
```

Comment:
immediately return the complete tuple, no need to waste performance for tuple concatenation:
```
return '--lines', str(licensecheck_lines), filename
```